torch/export/_remove_auto_functionalized_pass.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
@JacobSzwejbka it might be better to create a directory called `passes/` under export/ for these kind of things.

Reply:
I will do that in a follow up next Monday pinky promise